tidy3d/web/api/webapi.py

```diff
@@ -257,6 +264,7 @@ def upload(
             console.log(f"View task using web UI at [link={url}]'{url}'[/link].")
 
     remote_sim_file = SIM_FILE_HDF5_GZ
+    # for ModeSimulation, keep original simulation. User can manually reduce if needed.
```

Comment:
why?

Reply:
My reasoning was that the resulting sim_data.simulation will be whatever was uploaded to the server here, and it may be surprising if that is the reduced_simulation_copy. If the user wants to upload a smaller simulation, they would call the function reduced_simulation_copy explicitly, and then they would expect the sim_data to only know the reduced simulation.

open for discussion though

Reply:
Yeah, that makes sense! I’ll update the webapi then to do that